### DISCUSSION TITLE
How to generate OR condition with multiple conditions without a parenthesis on each one

### DISCUSSION THREAD

**Discussion by RangelReale** [go-jet/jet#223] [General]
```go
condition := Bool(true).OR(Bool(false)).OR(Bool(true)).OR(Bool(false))
```

```sql
((($2::boolean OR $3::boolean) OR $4::boolean) OR $5::boolean)
```

I tried various variations of putting the `.OR` code at different levels, but to no avail.

**Comment by go-jet** [comment]
You can use global OR function instead:

```golang
OR(
    Bool(true), 
    Bool(false), 
    Bool(true),
)
```

**Comment by RangelReale** [comment]
can I use this global OR directly in apply?

Em sáb., 11 de mar. de 2023 09:39, go-jet ***@***.***>
escreveu:

**Comment by go-jet** [comment]
I don't know what you mean by apply, but it can be used anywhere a bool expression is expected.
